src/chrome/content/rules/Fastmail.xml

```diff
@@ -27,8 +27,23 @@
 
 	<securecookie host="^\.www\.fastmail\.(?:com|fm)$" name=".+" />
 
-
-	<rule from="^http://((?:beta|qa|www)\.)?fastmail\.(com|fm)/"
-		to="https://$1fastmail.com/" />
+	<!--
+		The server sends an incorrect (non-EV) certificate when www. isn't used
```

Comment:
I'd prefer not to redirect between domains when the only difference is an EV cert vs a non-EV cert. So long as fastmail.com (sans www) answers HTTPS correctly, I think serving the EV cert on that domain is up to the site operator.

I do appreciate the test url additions, though! If you'd like to submit this ruleset as a straight rewrite without changing domain names.

Also, looking at the ruleset more closely it looks like the existing rule rewrites fastmail.fm to fastmail.com unnecessarily: fastmail.fm answers HTTPS correctly.